b2sdk/sync/folder_parser.py

```diff
@@ -12,7 +12,7 @@
 from .folder import B2Folder, LocalFolder
 
 
-def parse_sync_folder(folder_name, api):
+def parse_sync_folder(folder_name, api, local_folder_class=LocalFolder, b2_folder_class=B2Folder):
```

Comment:
Is there a reason for allowing users to provide their own classes? Something in TerraForm?

Reply:
No. Just apiver thing. By having dependency-injection pattern here, it is easier to make it apiver-complient

Reply:
I see. I like the idea